app/test_main.py

```diff
@@ -1,3 +1,19 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, result",
+    [
+        (0, 0, [0, 0]),  # test should return zero age
+        (14, 14, [0, 0]),  # test should return animals have not a year
+        (23, 23, [1, 1]),  # test should return animals have one year
+        (27, 27, [2, 2]),  # test should return animals have two years
+        (28, 28, [3, 2]),  # test should return animal have different age
+        (100, 100, [21, 17]),  # test should return animals have 100 years
```

Comment:
Add test case when `cat_age` and `dog_age` have different values to test that output would not be affected